wgpu/_classes.py

```diff
@@ -820,7 +820,8 @@ def create_buffer_with_data(self, *, label="", data, usage: "flags.BufferUsage")
                 includes bytes, bytearray, ctypes arrays, numpy arrays, etc.).
             usage (flags.BufferUsage): The ways in which this buffer will be used.
 
-        Alignment: the size (in bytes) of the data must be a multiple of 4.
+        Alignment: if the size (in bytes) of data is not a multiple of 4, the buffer
+                   size is rounded up to the nearest multiple of 4.
```

Comment:
I see that the info on alignment is missing in the docstring for `create_buffer()`. Let's add that while we're at it.

Reply:
Fixed